kalite/coachreports/static/js/coachreports/student_progress.js

```diff
@@ -101,10 +101,19 @@ var StudentProgressContainerView = Backbone.View.extend({
 
     initialize: function() {
         this.listenTo(this.collection, 'add', this.add_one);
+        
+        var self=this;
 
         this.render();
 
-        this.collection.fetch();
+        this.collection.fetch({
+                success: function() {
+                     if (self.collection.length == 0) {       //if the student visits the my progress page before attempting any quizes/videos
+                              show_message("info", gettext("Click on the LEARN button above to get started on your learning journey."));
+                              self.$el.html("");             //this is done to remove the 'Progress Report' header 
```

Comment:
Not really. We could leave the template here though, so that they know what the page is they landed on and why there is nothing there but a message.